supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -442,27 +436,6 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
     tellMetadata(metadataKeyValues)
   }
 
-  private def getEventMetadataOption(eventList: Seq[EventStartTime]): Option[Map[String, Any]] = {
```

Comment:
instead of the JES backend writing them all to the metadata, it still creates the list and lets the EJEA write them instead